### PR TITLE
Add option to set aria-label on step buttons

### LINKED ISSUE #540: Add option to set aria-label on step buttons
Need to add a label property in button configurations in steps config just like you have provided for cancel icon

### PR DESCRIPTION
**Pull request by lex111**
Resolves #540

### CODE REVIEW

**Comment by lex111** [comment]
@rwwagner90 done :heavy_check_mark:

File: test/unit/components/shepherd-footer.spec.js [hunk 1]
```diff
@@ -58,10 +58,12 @@ describe('components/ShepherdFooter', () => {
     expect(cancelButton).toHaveAttribute('tabindex', '0');
     expect(cancelButton).toHaveClass('shepherd-button-secondary cancel-button shepherd-button');
     expect(cancelButton).toHaveTextContent('Exit');
+    expect(cancelButton).toHaveAttribute('aria-label', 'Exit');
```
Comment (rwwagner90) [inline test/unit/components/shepherd-footer.spec.js]: Could you please add a separate test for labels? We should test what happens when nothing is set for label, when label is set to null, undefined, string, number, etc and handle all the cases.

Comment (lex111) [reply test/unit/components/shepherd-footer.spec.js]: Is that good enough for you?

Comment (rwwagner90) [reply test/unit/components/shepherd-footer.spec.js]: @lex111 perhaps I was not clear enough. I would like a separate test for labels please. Not adding an `expect` to the existing test, but a separate test to check label functionality. It should cover all the different cases I mentioned: what happens when nothing is set for label, when label is set to null, undefined, string, number, etc

Comment (lex111) [reply test/unit/components/shepherd-footer.spec.js]: @rwwagner90 Sorry, I really misunderstood.
Need a separate `.spec` file for cancel button test? Or how best to do in this case?

Comment (rwwagner90) [reply test/unit/components/shepherd-footer.spec.js]: @lex111 no, just a separate test in the same file please

Comment (rwwagner90) [reply test/unit/components/shepherd-footer.spec.js]: @lex111 I can add the tests, if you would like

Comment (lex111) [reply test/unit/components/shepherd-footer.spec.js]: @rwwagner90 sorry for the long answer.. yes, please add tests, I’m unlikely to do it soon.
